staging/kos/pkg/controllers/ipam/ipam.go

```diff
@@ -128,65 +124,86 @@ func NewController(netIfc kosclientv1a1.NetworkV1alpha1Interface,
 	eventIfc k8scorev1client.EventInterface,
 	queue k8sworkqueue.RateLimitingInterface,
 	workers int,
-	hostname string) *IPAMController {
+	hostname string,
+	metricsNamespace string,
+	metricsSubsystem string) *IPAMController {
```

Comment:
Only one value appears for `metricsNamespace` here, right?  Only one value appears for `metricsSubsystem` here, right?  This code knows its namespace and subsystem, it does not need to be told by the caller.